cli/crawl-cli.js

```diff
@@ -48,6 +50,16 @@ async function run(inputUrls, outputPath, verbose, logPath, numberOfCrawlers, da
         }
     };
 
+    /**
+     * @type {function(...any):string}
+     * @param {URL} url
+     */
+    const getPath = (url => {
```

Comment:
Nit - perhaps we could name this function a bit differently, current name suggests retrieving path portion of the provided url. How about something like `createOutputPath(url)` or `generateOutputPath(url)`?

Reply:
Sure can do, updated.  FWIW I think of these as getters and setters.